cloud-mgmt/config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+type Config struct {
+	HttpAddr      string      `json:"httpAddr"`
+	GrpcAddr      string      `json:"grpcAddr"`
+	MongoURI      string      `json:"mongoURI"`
+	Collections   interface{} `json:"collections"`
+	KafkaBroker   string      `json:"kafkaBroker"`
+	WebHookURL    string      `json:"webHookURL"`
+	WebHookSecret string      `json:"webHookSecret"`
+	ClientID      string      `json:"clientID"`
+	ClientSecret  string      `json:"clientSecret"`
+}
+
```

Comment:
add models for serializing / deserializing cluster-config.yaml to use for CAPI object.